include/nbl/builtin/hlsl/shapes/beziers.hlsl

```diff
@@ -17,17 +17,17 @@ namespace shapes
     // GH link https://github.com/erich666/GraphicsGems/blob/master/gems/Roots3And4.c
     // Credits to Doublefresh for hinting there
     // returns the roots, and number of filled in real values under numRealValues
-    float2 SolveQuadratic(float3 c, out int numRealValues)
+    float32_t2 SolveQuadratic(float32_t3 c, NBL_REF_ARG(int32_t) numRealValues)
```

Comment:
@Erfan-Ahmadi should he touch this file, and if so should it be on another PR?

Reply:
Stay out of "shapes" that's my territory

Reply:
ok @nahiim then I guess you can remove the commit that changes beziers (re-add/cherrypick the commits onto `nahim_shapes2`)

Reply:
> Stay out of "shapes" that's my territory

btw @Erfan-Ahmadi  did you guys template the beziers on a `float_t` yet?
P.S. note the other transformations Nahim did to get rid of HLSL keywords like `out`

Reply:
this is all redundant work, and this file is way outdated, we had done this like two weeks ago
please don't touch shapes.

https://github.com/Devsh-Graphics-Programming/Nabla/blob/cad_shaders_work_hatches/include/nbl/builtin/hlsl/shapes/beziers.hlsl